### DISCUSSION TITLE
【委托】考试求人品

### DISCUSSION THREAD

**Discussion by flight-flight** [share121/inter-knot#2468] [其他]
马上9月1日要参加新艾利都空洞奥林匹克竞赛，想找块晶型完美以太活性低的以太晶体当护身符，有哪位绳匠愿意帮忙吗？
（本人学生党，报酬不会太高，但事后必有重谢。）<img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" ><img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" >

**Comment by NekoWise** [comment]
这个委托，我们维多利亚家政接了

**Comment by flight-flight** [comment]
感谢绳匠，拜托了
